linux/tests/reals_linux/real_interlocked.h

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#ifndef REAL_INTERLOCKED_H
+#define REAL_INTERLOCKED_H
+
+#ifdef __cplusplus
+#include <cstdint>
+#include <cstddef>
+#else
+#include <stdint.h>
+#include <stddef.h>
+#endif
+
+#include "azure_macro_utils/macro_utils.h"
+
+#define R2(X) REGISTER_GLOBAL_MOCK_HOOK(X, real_##X);
+
+#ifdef _WIN64 

```

Comment:
I don't think we need this for Linux...

Reply:
"copy&paste is a design mistake" [Doc - approx 2007]

fixed

---
In reply to: [460143077](https://github.com/Azure/azure-c-pal/pull/16#discussion_r460143077) [](ancestors = 460143077)